packages/console/src/HeapUsage.tsx

```diff
@@ -168,17 +170,17 @@ function HeapUsage({
                 colorway: ['#4878ea'],
                 xaxis: {
                   dtick: Math.round(monitorDuration / 6),
-                  gridcolor: ChartTheme.linecolor,
+                  gridcolor: chartTheme?.linecolor,
                   range: [lastTimestamp - monitorDuration, lastTimestamp],
-                  linecolor: ChartTheme.linecolor,
+                  linecolor: chartTheme?.linecolor,
```

Comment:
Right now we're just not setting these values if no chart theme is provided. I think either we should just throw if no theme is provided (in `useChartTheme`), or automatically provide the default theme if no provider is set.

Reply:
Changed to required